### ISSUE TITLE
High vertical resolution with 0.25° horizontal resolution

### ISSUE THREAD

**Issue by wghuneke** [willaguiar/DSW-collaborative-project#28]
Run an experiment using OM2-025 but with the high-resolution vertical grid of OM2-01. This experiment is low priority but could be a nice addition to complete the set of vertical resolution tests and to show that a high horizontal resolution is (most likely) also needed.

**Comment by adele-morrison** [comment]
On this, it might be worth reviewing [Mensah et al. 2021](https://www.sciencedirect.com/science/article/pii/S1463500321000950) to see what they say about horizontal resolution requirements for overflows.

**Comment by willaguiar** [comment]
Reviving the issue, as this is one of the suggestions from the reviewers. 
___________
I am currently running ACCESS-OM2-025 with the same 1mtop (KDS75) and 5mtop (72 levels) used in the previous experiments with OM2-01. Once complete, the results will be posted here.


PS: @PaulSpence , these experiments might be useful in light of your plans for the PhD student w/ Madi to look at Ekman spirals in CMIP-like models ( similar horizontal resolution). I can direct you to the output folders once the simulations are complete.

**Comment by willaguiar** [comment]
Below is an early comparison for the first 3 years of the OM2-025 runs with 1mtop and 5mtop grids.


Essentially the DSW formation sensitivity (SWMT,a) to surface cell thickness is similar between OM2-01 and OM2-025. But as we expect, the overflow is weaker in OM2-025 likely due to excessive mixing.  DSW formation on OM2-025-1m [f]  ( defined by the same density definition we used in OM2-01) seem to highlight the right locations on Ross, Weddell, Adelie and Prydz.

These results highlight that the vertical grid could have an impact on DSW formation in CMIP6 models (mostly ~0.25 resolution), but overflow will still be a big limitation. 

PS: The timeseries of DSW formation seem to be more noisy tho [c].  I imagine that noisy timeseries in OM2-025 is because of its large surface cells (less cells but larger cells on the shelf, creating a more noisy signal)


![Image](https://github.com/user-attachments/assets/8fa59df8-6ee9-482c-9403-c2397dc1b3f9)

**Comment by willaguiar** [comment]
PS2: I guess it is important to highlight that for some reason, I can only run these simulations with OM2-025 and high vertical resolution  with really low time steps ( 10% of the time steps used in om2-025 regular set up). Im assuming here this is due to the high vertical resolution. If that is the case, then these simulations could still be somewhat prohibitive and expensive to run in climate timescales.

**Comment by AndyHoggANU** [comment]
Interesting — @wghuneke were just talking about an experiment like this yesterday - hadn’t realised you were already running it! Note that we can probably deal with the timestep issue with a bit of work. Will be interesting to see what happens after a few more years of simulation.

**Comment by willaguiar** [comment]
Yeah -  I was thinking that if the surface DSW formation persist after 10 years ( and the little overflow too), then it might be worth running a OM2-025 - 0.5m top ( and maybe with  CM2-025 too?).

out of context, the errors I get when I raise the timestep are temp or salinity errors like these below. depth/lon varies, but latitude is always is south of 30S:

`FATAL from PE   408:  Error: salinity out of range with value    -2.766293766776E+02 at (i,j,k) = (1437, 291,  1),  (lon,lat,dpt) = (   79.1250,  -45.9648,    0.5005 m)`

**Comment by willaguiar** [comment]
Here is how theses simulation look like after 8 years

The SWMT in the  0.25 and 0.1 models are rather similar (a,c). The overflow in OM025-1m is about 2/3rds weaker than OM01-1m, and seems to happen mostly around Ross and Weddell Seas(b,d, f). Yet, the ages in f suggest that although the overflow crosses the 1000 m isobath, just part of it gets to the Abyss ( from the Ross Sea). I’m guessing it could be worth having the OM2-025-1m running (or CM2) if we can activate the MOM5 overflow parameterizations in the Antarctic Shelf.
![Image](https://github.com/user-attachments/assets/cbaba76f-8bc4-46da-be6c-e6f1c2355a87)

**Comment by AndyHoggANU** [comment]
OK, very nice. Note that it might be possible that the overflow does make its way to the abyss, but that it mixes more so that the flow which gets to the bottom cell is diluted. Maybe transects of age + a vertical sum/average of age would give some info on that?

**Comment by adele-morrison** [comment]
I think the age plots above are already pretty clear evidence that the bottom water is mixing more in 1/4deg as it flows down the slope.

Note the higher SWMT in 1/4deg than 1/10th deg is consistent with what Christina found for panan. At low res, the eddies are less effective, so the isopycnals in the control are steeper. This means the surface waters on the shelf are preconditioned to be denser at lower res, so SWMT is more effective / stronger at low res. 

But there is more mixing at lower res as you go down the slope. Another way to show this would be to plot DSW overflow at 2500m isobath compared with 1000m isobath. I would expect the difference in transport with resolution (i.e. difference between light and dark blue lines in panel d above) to increase as you measure it further down the slope.
